src/main/java/bio/terra/pdao/bigquery/BigQueryPdao.java

```diff
@@ -365,27 +375,26 @@ public void addRowIdsToStagingTable(Dataset dataset, String stagingTableName) {
          * SET datarepo_row_id = GENERATE_UUID()
          * WHERE datarepo_row_id IS NULL
          */
+
         BigQueryProject bigQueryProject = bigQueryProjectForDataset(dataset);
-        StringBuilder sql = new StringBuilder();
-        sql.append("UPDATE ")
-            .append("`")
-            .append(bigQueryProject.getProjectId())
-            .append(".")
-            .append(prefixName(dataset.getName()))
-            .append(".")
-            .append(stagingTableName)
-            .append("` SET ")
-            .append(PDAO_ROW_ID_COLUMN)
-            .append(" = GENERATE_UUID() WHERE ")
-            .append(PDAO_ROW_ID_COLUMN)
-            .append(" IS NULL");
 
-        bigQueryProject.query(sql.toString());
+        String sqlTemplate =
```

Comment:
Same comment here. Perhaps I'm a bit defensive here, since I wrote the original code. IMHO, this change makes it harder to read the query and it does not appear to change the functionality of the method.